examples/conformance/CMakeLists.txt

```diff
@@ -0,0 +1,954 @@
+    "${TS_NAME}_images_samplerless_reads_2Darray_pitch"
+    "${TS_NAME}_integer_ops_1"
+    "${TS_NAME}_integer_ops_2"
+    "${TS_NAME}_integer_ops_3"
+    "${TS_NAME}_bruteforce"
+    "${TS_NAME}_mem_host_flags"
+    "${TS_NAME}_multiples"
+    "${TS_NAME}_printf"
+    "${TS_NAME}_profiling"
+    "${TS_NAME}_relationals"
+    "${TS_NAME}_relationals_shuffle"
+    "${TS_NAME}_select_uchar_uchar"
+    "${TS_NAME}_select_uchar_char"
+    "${TS_NAME}_select_char_uchar"
+    "${TS_NAME}_select_char_char"
+    "${TS_NAME}_select_ushort_ushort"
+    "${TS_NAME}_select_ushort_short"
+    "${TS_NAME}_select_short_ushort"
+    "${TS_NAME}_select_short_short"
+    "${TS_NAME}_select_uint_uint"
+    "${TS_NAME}_select_uint_int"
+    "${TS_NAME}_select_int_uint"
+    "${TS_NAME}_select_int_int"
+    "${TS_NAME}_select_ulong_ulong"
+    "${TS_NAME}_select_ulong_long"
+    "${TS_NAME}_select_long_ulong"
+    "${TS_NAME}_select_long_long"
+    "${TS_NAME}_thread_dimensions_quick"
+    "${TS_NAME}_thread_dimensions_full"
+    "${TS_NAME}_vecalign"
+    "${TS_NAME}_vecstep"
+  PROPERTIES
+    LABELS "conformance_suite"
+    ENVIRONMENT "POCL_LIMIT_MEMORY=1")
+
+
+set_tests_properties(
+    "${TS_NAME}_allocations_buffer"
+    "${TS_NAME}_allocations_image"
+    "${TS_NAME}_api"
+    "${TS_NAME}_atomics"
+    "${TS_NAME}_basic_math"
+    "${TS_NAME}_basic_s2v"
+    "${TS_NAME}_basic_memory"
+    "${TS_NAME}_basic_image"
+    "${TS_NAME}_basic_other"
+    "${TS_NAME}_buffers_read"
+    "${TS_NAME}_buffers_write"
+    "${TS_NAME}_buffers_fill"
+    "${TS_NAME}_buffers_var1"
+    "${TS_NAME}_buffers_var2"
+    "${TS_NAME}_buffers_map_read"
+    "${TS_NAME}_buffers_map_write"
+    "${TS_NAME}_commonfns"
+    "${TS_NAME}_compiler"
+    "${TS_NAME}_computeinfo"
+    "${TS_NAME}_contractions"
+    "${TS_NAME}_conversion_ushort_uchar"
+    "${TS_NAME}_conversion_uint_uchar"
+    "${TS_NAME}_conversion_float_uchar"
+    "${TS_NAME}_conversion_short_char"
+    "${TS_NAME}_conversion_int_char"
+    "${TS_NAME}_conversion_float_char"
+    "${TS_NAME}_conversion_uchar_ushort"
+    "${TS_NAME}_conversion_uint_ushort"
+    "${TS_NAME}_conversion_float_ushort"
+    "${TS_NAME}_conversion_char_short"
+    "${TS_NAME}_conversion_int_short"
+    "${TS_NAME}_conversion_float_short"
+    "${TS_NAME}_conversion_uchar_uint"
+    "${TS_NAME}_conversion_ushort_uint"
+    "${TS_NAME}_conversion_float_uint"
+    "${TS_NAME}_conversion_char_int"
+    "${TS_NAME}_conversion_short_int"
+    "${TS_NAME}_conversion_float_int"
+    "${TS_NAME}_conversion_char_float"
+    "${TS_NAME}_conversion_short_float"
+    "${TS_NAME}_conversion_int_float"
+    "${TS_NAME}_conversion_char_double"
+    "${TS_NAME}_conversion_short_double"
+    "${TS_NAME}_conversion_int_double"
+    "${TS_NAME}_conversion_long_double"
+    "${TS_NAME}_conversion_uchar_ulong"
+    "${TS_NAME}_conversion_ushort_ulong"
+    "${TS_NAME}_conversion_uint_ulong"
+    "${TS_NAME}_conversion_float
```

Comment:
I see you added all tests separately to Cmake. Is there a chance that new tests are added to the conformance test suite without us noticing them? Can you add a test for that?